src/timer.js

```diff
@@ -0,0 +1,55 @@
+import { switcher } from "./switch.js";
+import { Howl } from "howler";
+const timerStop = document.getElementById("timer_stop");
+const timeOutput = document.getElementById("time_output");
+const timeInput = document.getElementById("time_input");
+
+timerTitle.addEventListener("click", switcher);
+
+var sound = new Howl({
+    src: [require("./sounds/sound.mp3")]
+});
```

Comment:
У тебя наверное поэтому звук не работает, не передается путь до звука в сборку
Во первых, чтобы webpack вообще включил в сборку твой файл со звуком, нужно его импортировать в джаваскрипте
Во вторых путь до файла в сборке нужно передать в Howl, путь до файлика в сборке возвращается вебпаком из 'импорта' 
```suggestion

import soundSrc from './путь_до_звука';

var sound = new Howl({
    src: soundSrc
});
```